src/main/java/com/exasol/errorcodecrawlermavenplugin/crawler/MitigationStepReader.java

```diff
@@ -10,22 +11,27 @@
 import spoon.reflect.code.CtInvocation;
 
 /**
- * Reader for invocations of {@link ErrorMessageBuilder#mitigation(String)}.
+ * Reader for invocations of {@link ErrorMessageBuilder#mitigation(String, Object...)}.
  */
 class MitigationStepReader implements MessageBuilderStepReader {
-    private static final String SIGNATURE = "mitigation(java.lang.String)";
+    private static final Set<String> SUPPORTED_SIGNATURE = Set.of("mitigation(java.lang.String,java.lang.Object[])",
+            "mitigation(java.lang.String)");
 
     @Override
     public void read(final CtInvocation<?> builderCall, final ErrorMessageDeclaration.Builder errorCodeBuilder,
             final Path projectDirectory) throws InvalidSyntaxException {
         final List<CtExpression<?>> arguments = builderCall.getArguments();
-        assert arguments.size() == 1;
+        assert !arguments.isEmpty();
         final CtExpression<?> messageArgument = arguments.get(0);
-        errorCodeBuilder.prependMitigation(new ArgumentReader(SIGNATURE).readStringArgumentValue(messageArgument));
+        final String mitigation = new ArgumentReader(builderCall.getExecutable().getSignature())
+                .readStringArgumentValue(messageArgument);
+        errorCodeBuilder.prependMitigation(mitigation);
+        new DirectParameterReader().readInlineParameters(arguments.size() - 1, mitigation, errorCodeBuilder);
     }
 
     @Override
     public boolean canRead(final String className, final String methodSignature) {
-        return className.equals(ErrorMessageBuilder.class.getSimpleName()) && methodSignature.equals(SIGNATURE);
+        return className.equals(ErrorMessageBuilder.class.getSimpleName())
```

Comment:
this implementation of this method is the same for both `MitigationStepReader` and `MessageStepReader`.
Is this on purpose? if so, it's on purpose to have this duplication?

Reply:
There's also no test for this condition

Reply:
it's tested when crawling a method call